src/Bicep.Decompiler/BicepHelpers/SyntaxHelpers.cs

```diff
@@ -1,132 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text.RegularExpressions;
-using Bicep.Core.Diagnostics;
-using Bicep.Core.Extensions;
 using Bicep.Core.Parsing;
 using Bicep.Core.Syntax;
 
 namespace Bicep.Decompiler.BicepHelpers
 {
     public static class SyntaxHelpers
```

Comment:
There's the `TestSyntaxFactory` class that can also be merged into `Bicep.Core.Syntax.SyntaxFactory`, but that one has too many references. I'll send a separate refactoring PR to do that.